mne/channels/channels.py

```diff
@@ -1173,9 +1174,238 @@ def _recursive_flatten(cell, dtype):
+        fname='KIT-208_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='KIT-NYU-2019',
+        description='',
+        fname='KIT-NYU-2019_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='KIT-UMD-1',
+        description='',
+        fname='KIT-UMD-1_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='KIT-UMD-2',
+        description='',
+        fname='KIT-UMD-2_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='KIT-UMD-3',
+        description='',
+        fname='KIT-UMD-3_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='KIT-UMD-4',
+        description='',
+        fname='KIT-UMD-4_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='neuromag306mag',
+        description='',
+        fname='neuromag306mag_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='neuromag306planar',
+        description='',
+        fname='neuromag306planar_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='neuromag122cmb',
+        description='',
+        fname='neuromag122cmb_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='neuromag306cmb',
+        description='',
+        fname='neuromag306cmb_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='ecog256',
+        description='',
+        fname='ecog256_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='ecog256bipolar',
+        description='',
+        fname='ecog256bipolar_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='eeg1010_neighb',
+        description='',
+        fname='eeg1010_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='elec1005',
+        description='',
+        fname='elec1005_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='elec1010',
+        description='',
+        fname='elec1010_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='elec1020',
+        description='',
+        fname='elec1020_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='itab28',
+        description='',
+        fname='itab28_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='itab153',
+        description='',
+        fname='itab153_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='language29ch-avg',
+        description='',
+        fname='language29ch-avg_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='mpi_59_channels',
+        description='',
+        fname='mpi_59_channels_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='yokogawa160',
+        description='',
+        fname='yokogawa160_neighb.mat',
+    ),
+    _BuiltinChannelAdjacency(
+        name='yokogawa440',
+        description='',
+        fname='yokogawa440_neighb.mat',
+    ),
+]
+
+
+@fill_doc
+def get_builtin_ch_adjacencies():
+    """Get a list of all FieldTrip neighbor definitions shipping with MNE.
```

Comment:
```suggestion
    """Get a list of all neighbor definitions shipping with MNE.
```

Reply:
Currently it's only FT neighbors, and I don't want to alter this for now. It's always only been FT neighbors in the past, and we **only** support FT's `.mat` format so far anyway.